Marlin/src/gcode/feature/ft_motion/M493.cpp

```diff
@@ -423,15 +416,8 @@ void GcodeSuite::M493() {
 
   #endif // HAS_Y_AXIS
 
-  planner.synchronize();
-
   if (flag.update) ftMotion.update_shaping_params();
 
-  if (flag.reset_ft) {
-    stepper.ftMotion_syncPosition();
-    ftMotion.reset();
-  }
```

Comment:
Are we certain that changing any of the above parameters does/will not require `ftMotion.reset()` to recalculate something/anything based on the updated parameter(s)?